app/assets/stylesheets/utilities/_overwrites.scss

```diff
@@ -92,3 +92,11 @@
 .govuk-grid-column-two-thirds {
   position: relative;
 }
+
+.govuk-tabs {
+  .govuk-heading-l {
+    @include govuk-media-query($from: tablet) {
+      @include govuk-visually-hidden;
+    }
+  }
+}
```

Comment:
Is this change something we might want to push into the component to remove our overwrite need?

Reply:
Unfortunately there's no general rule for this - it depends on the usecase. For us it doesn't make sense to have the tab title appear again the tab container, but for others (e.g. content data last time I checked) were showing the title within the tabs. It feels to early for me to move this upstream, but it's an option for sure.